slf4j-jdk-platform-logging/src/main/java/org/slf4j/jdk/SLF4JSystemLoggerFinder.java

```diff
@@ -0,0 +1,34 @@
+package org.slf4j.jdk;
+
+import org.slf4j.LoggerFactory;
+
+/**
+ * Uses SLF4J's {@link LoggerFactory#getLogger(String)} to get a logger
+ * that is adapted for {@link System.Logger}.
+ */
+public class SLF4JSystemLoggerFinder extends System.LoggerFinder {
+
+    @Override
+    public System.Logger getLogger(String name, Module module) {
+        // JEP 264[1], which introduced the Platform Logging API,
+        // contains the following note:
+        //
+        //  > An implementation of the LoggerFinder service should make it
+        //  > possible to distinguish system loggers (used by system classes
+        //  > from the Bootstrap Class Loader (BCL)) and application loggers
+        //  > (created by an application for its own usage). This distinction
+        //  > is important for platform security. The creator of a logger can
+        //  > pass the class or module for which the logger is created to the
+        //  > LoggerFinder so that the LoggerFinder can figure out which kind
+        //  > of logger to return.
+        //
+        // If backends support the distinction support this distinction and
+        // once `LoggerFactory`'s API is updated to forward a module, we
+        // should do that here.
+        //
+        // [1] https://openjdk.java.net/jeps/264
```

Comment:
Does this make sense?

Reply:
I'm not sure that I quite get the point: the `System.Logger` API is intended for JDK classes, not for applications, or is it?

The javadoc of `System.LoggerFinder` requires a slightly different thing:

```
     * In addition, when a security manager is present, loggers provided to
     * system classes should not be directly configurable through the logging
     * backend without requiring permissions.
     * <br>
     * It is the responsibility of the provider of
     * the concrete {@code LoggerFinder} implementation to ensure that
     * these loggers are not configured by untrusted code without proper
     * permission checks, as configuration performed on such loggers usually
     * affects all applications in the same Java Runtime.
```

We don't know the logging backends and how they configure things. So we have to check with the `SecurityManager` for _all_ cases, don't we? But this is already checked by the constructor of the `System.LoggerFinder`... I'm confused.

Reply:
> I'm not sure that I quite get the point: the `System.Logger` API is intended for JDK classes, not for applications, or is it?

It will be used by JDK classes, but it's implemented here and the quote addresses implementations, so I assume it applies to us. Didn't think to look at the JavaDoc, though, and it's not mentioned there, so maybe it's outdated?

Re security manager, it is indeed weird that `System.LoggerFinder` already does the check. Not sure what to do here.